tests/vxingest/netcdf_to_cb/test_int_metar_obs_netcdf.py

```diff
@@ -34,19 +35,35 @@ def stub_worker_log_configurer(queue: Queue):
 def setup_connection():
     """test setup"""
     _vx_ingest = VXIngest()
-    _vx_ingest.credentials_file = (os.environ["CREDENTIALS"],)
+    # Ensure credentials_file is a string, not a tuple
+    credentials = os.environ["CREDENTIALS"]
+    if isinstance(credentials, tuple):
+        credentials = credentials[0]
+    _vx_ingest.credentials_file = credentials
     _vx_ingest.cb_credentials = _vx_ingest.get_credentials(_vx_ingest.load_spec)
     _vx_ingest.connect_cb()
-    _vx_ingest.load_spec["ingest_document_ids"] = _vx_ingest.collection.get(
-        "JOB-TEST:V01:METAR:NETCDF:OBS"
-    ).content_as[dict]["ingest_document_ids"]
+    try:
+        vx_ingest = setup_connection(_vx_ingest)
+        id_query = """DELETE
+                FROM `vxdata`.`_default`.`METAR` f
+                WHERE f.subset = 'METAR'
+                AND f.type = 'DF'
+                AND f.url LIKE '/opt/data/%' RETURNING f.id AS id;"""
+        row_iter = vx_ingest.cluster.query(
+            id_query, QueryOptions(metrics=True, read_only=False)
+        )
+        for row in row_iter:
+            print(f"Deleted {row['id']}")
```

Comment:
Is there a reason to be deleting docs in test setup? This sounds like it could destroy `DF` documents we want to keep.

Reply:
notice the ..AND f.url LIKE '/opt/data/%. - this indicates that it is test data. When I said real ingest I meant the real process getting run against the test directory.